src/include/storage/arrow_block_metadata.h

```diff
@@ -0,0 +1,241 @@
+#pragma once
+#include <map>
+#include <unordered_set>
+#include "storage/block_layout.h"
+#include "storage/storage_defs.h"
+#include "storage/storage_util.h"
+
+namespace terrier::storage {
+
+// TODO(Tianyu): Can use to specify cases where we don't concat per-block in the future (e.g. no need to put
```

Comment:
Is this mostly a TODO for yourself? I can't imagine anyone else will know what this means.

Reply:
Sorry. Rewritten this to be more clear.